### PR TITLE
Add Clowder deployment template

### PR DESCRIPTION
**Pull request by Victoremepunto**
Provides a Clowder template for deployments in Clowder managed Openshift namespaces

### CODE REVIEW

File: clowdapp.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+objects:
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: floorist
+  spec:
+    database:
+      sharedDbAppName: ${SHARED_DB_APP_NAME}
+    objectStore:
+      - ${BUCKET_NAME}
```
Comment (skateman) [inline clowdapp.yaml:15]: As discussed, we can't have the prod bucket provided via clowder.

File: clowdapp.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    envName: ${ENV_NAME}
+    jobs:
+    - name: metrics-exporter
+      schedule: '*/5 * * * *'
```
Comment (skateman) [inline clowdapp.yaml:19]: This should be parameterized, so each project can have its own schedule. Ideally the variable name should be unique enough in *that* big bad repo.

File: clowdapp.yaml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+          value: "from a secret"
+        - name: AWS_REGION
+          value: "from a secret"
+        - name: AWS_BUCKET
+          value: "from a secret"
+        - name: AWS_ENDPOINT
+          value: "from a secret"
+            
+        - name: FLOORPLAN_FILE
+          value: "from a secret"
```
Comment (skateman) [inline clowdapp.yaml:36]: No need to be a secret, or maybe not even a dynamic parameter, we can just make this something hardcoded and mount the configmap accordingly.

File: clowdapp.yaml (hunk 2 above)
Comment (Victoremepunto) [reply clowdapp.yaml]: I've added a parameter to the template with the surprising name of `SCHEDULE` to configure the cronjob's schedule. However I don't think the variable name needs to be unique - the cronjob definition is supposed to be unique on each Insights app namespace.

File: clowdapp.yaml (hunk 1 above)
Comment (Victoremepunto) [reply clowdapp.yaml:15]: I've added a configuration system on #10  to allow to force read the bucket name from the environment even on Clowder managed deployments, that should work. if it doesn't I think we can create a second template that doesn't have the objectStore definition.

File: clowdapp.yaml (hunk 3 above)
Comment (Victoremepunto) [reply clowdapp.yaml]: Yeah - this was an early WIP - I changed this to read from a ConfigMap

File: clowdapp.yaml (hunk 1 above)
Comment (skateman) [reply clowdapp.yaml:17]: :+1:

**Review by skateman — APPROVED** [review]
The Seal of Approval

**Pull request merged**